dockerfiles/Dockerfile.dev

```diff
@@ -4,6 +4,17 @@
 
 FROM pysyft-base:local
 
+RUN apk update
```

Comment:
Using update (apt or apk) in Docker files is highly discouraged, you can check on their guidelines for a full explanation, but in short it has to do with the fact that Docker caches the intermediate containers simply based on the instruction string. Can you put together those commands using `&&`?

Reply:
It also adds extra bloat to the image.  I'd just recommend doing `apk add --initdb --no-cache git redis`.

Reply:
Done.